appnav/src/main/kotlin/io/element/android/appnav/room/joined/JoinedRoomLoadedFlowNode.kt

```diff
@@ -169,7 +179,7 @@ class JoinedRoomLoadedFlowNode @AssistedInject constructor(
 
     sealed interface NavTarget : Parcelable {
         @Parcelize
-        data object Messages : NavTarget
+        data class Messages(val eventId: EventId? = null) : NavTarget
```

Comment:
We should find a better name than `eventId` as it's a bit obscure.
Maybe `focusedEventId` or`permalinkEventId`?

Reply:
We had `targetEventId` (and `highlightEventId`), but maybe `focusedEventId` will work since it's closed to the SDK API.

Reply:
Done